test/duplicates-ind1.spec.js

```diff
@@ -92,36 +92,4 @@ describe('duplicates-ind1', () => {
 			expect(result).to.eql({valid: false});
 		});
 	});
-
```

Comment:
This file should not be modified in this PR.

Reply:
fixed